js/tiles.js

```diff
@@ -303,6 +303,13 @@ const door = {
             this.ground = this.createImage("tiles/rooms/floor/live.svg");
             alertInfo("More to Go !", "");
         },
+    }),
+       design: Object.assign({}, OpenDoors, {
+        createImages: function() {
+            this.wallTop = this.createImage("tiles/rooms/door/top.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/right.svg");
+            this.ground = this.createImage("tiles/rooms/floor/handarjun.svg");
```

Comment:
Would you like to create the ground tile before the others? then maybe it is not overlapping the wall.

Reply:
sure will do